site/sidebar.md

```diff
@@ -64,10 +64,6 @@
     - [ResourceList](reference/schema/resource-list/)
     - [CRD Status Convention](reference/schema/crd-status-convention/)
     - [Config Connector Status Convention](reference/schema/config-connector-status-convention/)
-  - [Annotations Reference](reference/annotations/)
```

Comment:
This shouldn't be removed. This is a generated file. Looks like `porch` branch doesn't have the sidebar template updated.

Reply:
Oh, I see it now. In that case, may be merging this PR first sounds reasonable.

Reply:
Thanks